lib/plugins/inspec-plugin-manager-cli/lib/inspec-plugin-manager-cli/cli_command.rb

```diff
@@ -229,6 +229,7 @@ def install_from_path__apply_entry_point_heuristics(path) # rubocop: disable Met
         given = given.expand_path # Resolve any relative paths
         name_regex = /^(inspec|train)-/
         versioned_regex = /^(inspec|train)-[a-z0-9\-\_]+-\d+\.\d+\.\d+$/
+        sha_ref_regex = /^(inspec|train)-[a-z0-9\-\_]+-[0-9a-f]{5,40}$/
```

Comment:
I could make a single regex that matches either of these 2 regexes.
That would keep the lines changed below simpler; but might make the regex(es) unreasonable to follow.